test/test_client.py

```diff
@@ -143,6 +143,18 @@ def verify_init_args(api_instance, **kwargs):
         Client(api_class=MockApi, api_config_class=MockApiConfig, url='http://foo',
                account='myacct', login_id='mylogin', ca_bundle='mybundle')
 
+    def test_client_does_not_override_apiconfig_values_with_empty_values(self):
+        class MockApi(MockApiHelper):
+            def verify_init_args(api_instance, **kwargs):
+                api_instance.verify_apiconfig_dict_in(self, **kwargs)
+                self.assertEqual(kwargs['account'], 'apiconfigaccount')
+                self.assertEqual(kwargs['url'], 'apiconfigurl')
+                self.assertEqual(kwargs['ca_bundle'], 'apiconfigcabundle')
```

Comment:
should the `login_id` also be checked here?

Reply:
Good catch. Fixing.